docs/core/introduction.md

```diff
@@ -6,404 +6,52 @@ ms.date: 02/24/2022
 ms.custom: "updateeachrelease"
 recommendations: false
 ---
-# What is .NET? Introduction and overview
+# What is .NET?
 
-.NET is a free, open-source development platform for building many kinds of apps, such as:
+.NET is a free and open-source development platform for building any type of [app](tutorials/with-visual-studio-code.md). .NET is built on a [high-performance runtime](https://devblogs.microsoft.com/dotnet/category/performance/) that is used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/app/). It is built with the community on [GitHub](https://github.com/dotnet/core).
```

Comment:
"platform for building any type of app" is our north star, but I am not sure whether it is a fair statement to make.  I like more the wording used on https://dotnet.microsoft.com/en-us/learn/dotnet/what-is-dotnet .

Reply:
```suggestion
NET is a free, cross-platform, open source developer platform for building many different types of [applications](tutorials/with-visual-studio-code.md). .NET is built on a [high-performance runtime](https://devblogs.microsoft.com/dotnet/category/performance/) that is used in production by many [high-scale apps](https://devblogs.microsoft.com/dotnet/category/app/). It is built with the community on [GitHub](https://github.com/dotnet/core).
```